Adafruit_MQTT_FONA.h

```diff
@@ -42,7 +42,13 @@ class Adafruit_MQTT_FONA : public Adafruit_MQTT {
                      const char *user = "", const char *pass = "")
       : Adafruit_MQTT(server, port, user, pass), fona(f) {}
 
-  bool connectServer() {
+  bool connected() {
+    // Return true if connected, false if not connected.
+    return fona->TCPconnected();
+  }
+
+protected:
+  bool connectServer() override {
```

Comment:
Why are you appending `override` here? The base function's signature for `connectServer()` seems to be the same.

Reply:
The purpose of the `override` keyword is so that the compiler will issue a warning/error of the two signatures do not match. If, hypothetically, `Adafruit_MQTT::connectServer` was changed to `virtual int connectServer()` then this version would issue an error, but without the `override` keyword the base class version of connectServer would silently become it's own function without overriding `Adafruit_MQTT::connectServer`.  So it makes the code a bit less fragile. That being said, the base class is a **pure virtual** declaration, so there is less benefit to adding this. There's a [StackOverflow question](https://stackoverflow.com/questions/46446652/is-there-any-point-in-using-override-when-overriding-a-pure-virtual-function/46448798) discussing this. I'm happy to remove it if you prefer.

Reply:
Nope, it's fine. I learned something new! Tests OK on hardware, LGTM.